internal/mikrotik/provider_test.go

```diff
@@ -324,14 +324,24 @@ func TestContains(t *testing.T) {
 	}
 }
 
-func TestCleanupChanges(t *testing.T) {
+func TestChanges(t *testing.T) {
+	mikrotikProvider := &MikrotikProvider{
+		client: &MikrotikApiClient{
+			&MikrotikDefaults{},
+			nil,
+			nil,
+		},
+	}
+
```

Comment:
I think we should initialize a defaultTTL here so that we can test it being configurable

```suggestion
	defaultTTL := 1800
	mikrotikProvider := &MikrotikProvider{
		client: &MikrotikApiClient{
			&MikrotikDefaults{int64(defaultTTL)},
			nil,
			nil,
		},
	}

```